agent/metrics/v1/metrics_service.proto

```diff
@@ -16,9 +16,9 @@ syntax = "proto3";
 
 package opentelemetry.proto.agent.metrics.v1;
 
-import "opentelemetry/proto/agent/common/v1/common.proto";
-import "opentelemetry/proto/metrics/v1/metrics.proto";
-import "opentelemetry/proto/resource/v1/resource.proto";
+import "agent/common/v1/common.proto";
+import "metrics/v1/metrics.proto";
+import "resource/v1/resource.proto";
```

Comment:
If these files will be built with bazel then imported in C++ or other proto files will be strange to have them included as "metrics/..." it is very generic path. 

What is the motivation to move the files?

Reply:
No reason. At least java files use correct package name `io.opentelemetry` I am not sure what the build for other languages produces. 

I have reverted the path.